builtin/providers/postgresql/provider.go

```diff
@@ -35,6 +35,12 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("POSTGRESQL_PASSWORD", nil),
 				Description: "Password for postgresql server connection",
 			},
+			"sslmode": &schema.Schema{
```

Comment:
I think my gut feeling is that it's better for it to be `ssl_mode` to match with the style used elsewhere in Terraform. That way users won't need to refer to the docs to remember what style is used for this attribute.

(Sorry for the delay in answering.)